### ISSUE TITLE
Incompatible with Makeinfo 6.5

### ISSUE THREAD

**Issue by rpgoldman** [nikodemus/sb-texinfo#5]
I am unable to format sb-texinfo's manual with the Texinfo 6.5 toolchain, either as PDF or HTML:
```
This is `epsf.tex' v2.7.4 <14 February 2011>
) localization, formatting, and turning on texinfo input format.)
./sb-texinfo.texinfo:14: @setcontentsaftertitlepage has been removed as a Texin
fo command; move your @contents command if you want the contents after the titl
e page..
@setcontentsaftertitlepage ...ter the title page.}
                                                  
l.14 @setcontentsaftertitlepage
                               
? X
No pages of output.
```
and 
```
 $ make
makeinfo --html --no-split --css-include=style.css sb-texinfo.texinfo
sb-texinfo.texinfo:14: warning: @setcontentsaftertitlepage is obsolete; move your @contents command if you want the contents after the title page
include/sb-texinfo.texinfo:6: bad name for @macro
include/sb-texinfo.texinfo:9: bad name for @macro
include/sb-texinfo.texinfo:12: bad name for @macro
include/sb-texinfo.texinfo:15: bad name for @macro
include/sb-texinfo.texinfo:18: bad name for @macro
include/fun-sb-texinfo-document-package.texinfo:3: unexpected @
include/fun-sb-texinfo-document-package-pathname.texinfo:3: unexpected @
include/fun-sb-texinfo-document-package-pathname.texinfo:3: unexpected @
include/fun-sb-texinfo-generate-includes.texinfo:3: unexpected @
make: *** [sb-texinfo.html] Error 1
```
I will see if I can make a PR for this.

**Comment by rpgoldman** [comment]
OK, I found the answer to the other issue, as well, but I'm not sure how to fix it.  Here's a description of allowable Macro names from the most recent (6.6) Texinfo manual:

> For a macro to work consistently with TeX, macroname must consist entirely of letters: no digits, hyphens, underscores, or other special characters. So, we recommend using only letters. However, makeinfo will accept anything consisting of alphanumerics, and (except as the first character) ‘-’. 

These macro definitions have names that Texinfo now rejects as incorrect:
```
@ifnottex
@macro &allow-other-keys
&allow-other-keys
@end macro
@macro &optional
&optional
@end macro
@macro &rest
&rest
@end macro
@macro &key
&key
@end macro
@macro &body
&body
@end macro
@end ifnottex
```
For the ASDF manual, we had to change these to:
```
@macro AallowOtherKeys
&allow-other-keys
@end macro
@macro Aoptional
&optional
@end macro
@macro Arest
&rest
@end macro
@macro Akey
&key
@end macro
@macro Abody
&body
@end macro
```
That will require further editing of docstrings.lisp, etc. so I'd prefer to get a clear direction before providing a PR.

**Comment by rpgoldman** [comment]
This issue fixed by PR #9
